webapp/components/integrations/components/add_command.jsx

```diff
@@ -39,6 +50,10 @@ export default class AddCommand extends React.Component {
         this.updateAutocompleteHint = this.updateAutocompleteHint.bind(this);
         this.updateAutocompleteDescription = this.updateAutocompleteDescription.bind(this);
 
+        this.action = this.props.location.query.action;
```

Comment:
I see you're extending the add_command.jsx component to also be used for editing commands. What are your thoughts on moving it out to a separate component?

Maybe @hmhealey can chime in because he built these pages initially, but I think it would be better to have separate, single-purpose components rather than trying to force reuse out of one like this